RemotingSample/Assets/Prefabs/RuntimeInfo.prefab

```diff
@@ -0,0 +1,358 @@
+%YAML 1.1
```

Comment:
Do we want to have a definition of this prefab?

I thought the plan was to use the version of this from the SamplesShared package

You might already be using that shared one, and you just don't need this extra prefab

Reply:
I have the exact same comment about UnityLoggingPanel.prefab

Reply:
fair point, looks like we have to clean up some code in main test project too. I kept them as main project had it as well, even though it references samplesshared package. Removed them.